### PR TITLE
Fix golint failures of pkg/kubelet/apis

### PR DESCRIPTION
**Pull request by SataQiu**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

Fix golint failures of pkg/kubelet/apis

**Which issue(s) this PR fixes**:

Ref #68026

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by mattjmcnaughton — CHANGES_REQUESTED** [review]
Thanks for your pr :)

It looks like the `verify-golint` check is still failing bc of:

```
Errors from golint:
pkg/kubelet/apis/well_known_annotations_windows.go:30:2: exported const HypervIsolationValue should have comment (or a comment on this block) or be unexported
```
Otherwise, lgtm. Will add lgtm label after the final error is correct.

**Review by mattjmcnaughton — APPROVED** [review]
/lgtm

thanks for your work on these lint errors :)

**Review by fejta — COMMENTED** [review]
/approve

**Comment by SataQiu** [comment]
Can you review this PR? @yujuhong

**Review by fejta — COMMENTED** [review]
/lgtm
/approve

**Comment by SataQiu** [comment]
Can you review this PR? @yujuhong @Random-Liu

File: pkg/kubelet/apis/well_known_labels.go [hunk 1]
```diff
@@ -27,7 +27,10 @@ const (
 	// The OS/Arch labels are promoted to GA in 1.14. kubelet applies both beta
 	// and GA labels to ensure backward compatibility.
 	// TODO: stop applying the beta OS/Arch labels in Kubernetes 1.18.
-	LabelOS   = "beta.kubernetes.io/os"
+
+	// LabelOS is a label to indicate the operating system of the node.
```
Comment (liggitt) [inline pkg/kubelet/apis/well_known_labels.go]: Keep the comments about backward compatibility and the release they will no longer be applied associated with the variables

Comment (SataQiu) [reply pkg/kubelet/apis/well_known_labels.go]: Thank you very much! @liggitt 
I have updated the code.

**Pull request merged**